tests/benchmarks/array_spans_stress.php

```diff
@@ -0,0 +1,76 @@
+<?php
+// phpcs:disable PSR1.Files.SideEffects
+
+putenv('DD_TRACE_CLI_ENABLED=true');
+putenv('DD_SPANS_LIMIT=9999999');
+putenv('DD_AGENT_HOST=ddagent_integration');
+putenv('DD_TRACE_DEBUG_CURL_OUTPUT=0');
+putenv("DD_TRACE_DEBUG_CURL_OUTPUT=0");
+putenv('DD_TRACE_BETA_SEND_TRACES_VIA_THREAD=1');
+putenv('DD_TRACE_AGENT_TIMEOUT=500');
+putenv('DD_TRACE_AGENT_CONNECT_TIMEOUT=500');
+
+function_exists('dd_trace_internal_fn') && dd_trace_internal_fn('ddtrace_reload_config');
```

Comment:
Is there a reason `dd_trace_internal_fn` would not exist?

Reply:
I noticed later in the test it isn't guarded. They should either all be guarded, or none of them should, I think.